HubbleHW.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Check issue on P1 for better docstrings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Hubble/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>